.env.example

```
@@ -105,6 +105,22 @@ SHEETS_TIMEZONE=America/Denver
 SHEETS_SYNC_INTERVAL=900000
 CRON_SECRET=your-cron-secret-for-vercel
 
+# ================================================
+# REGISTRATION SYSTEM
+# ================================================
+# JWT signing secret for registration tokens (min 32 chars)
+REGISTRATION_JWT_SECRET=your-secure-registration-jwt-secret-min-32-chars
+# Token expiry duration (default: 24h)
+REGISTRATION_TOKEN_EXPIRY=86400000
+# Registration deadline (hours after purchase)
+REGISTRATION_DEADLINE_HOURS=72
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The REGISTRATION_DEADLINE_HOURS key should go before the REGISTRATION_JWT_SECRET key

(UnorderedKey)